galite-core/src/main/kotlin/org/kopi/galite/domain/exceptions/InvalidValueException.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2013-2020 kopiLeft Services SARL, Tunis TN
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1 as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package org.kopi.galite.domain.exceptions
+
+/**
+ * Thrown to indicate that an invalid value has been passed to a field.
+ *
+ * @param value to pass
+ * @param label the field's label
+ *
+ */
+class InvalidValueException(value: Comparable<*>, label: String) : IllegalArgumentException() {
```

Comment:
Why use IllegalArgumentException? I recommend using RuntimeException. Because it is not an error caused by a method that received an illegal or inappropriate argument